odoo/addons/base/views/res_partner_views.xml

```diff
@@ -174,9 +174,9 @@
 
                     <group>
                         <group>
-                            <span class="o_form_label o_td_label" name="address_name">
+                            <span class="o_form_label o_td_label o_address_type" name="address_name">
```

Comment:
can we get rid of this new classname?

Reply:
Not really @aab-odoo , we need to bind the css to this element somehow.
Since its siblings comes with the `o_address_format ` class already, it make sense to me add this new class.
(fyi: mano is currently  off, I'm handling this)